InstallCommands.php

```diff
@@ -34,7 +34,7 @@ public function install() {
     }
 
     FileCacheFactory::setConfiguration(['default' => ['class' => '\Drupal\Component\FileCache\NullFileCache']]);
-    $source_storage = new FileStorage(config_get_config_directory(CONFIG_SYNC_DIRECTORY));
+    $source_storage = new FileStorage(Settings::get('config_sync_directory'));
```

Comment:
I think I would use `drush_config_get_config_directory` here but 8.7.x becomes end of life in six days so this change is probably fine.